src/index.js

```diff
@@ -42,9 +40,12 @@ class SmartChart extends React.Component {
 
     render() {
         const {children, isMobile} = this.props;
+
+        this.chart.setIsMobile(this.props.isMobile);
```

Comment:
@shayanypn do not set state in render() method.
That's totally frowned up on in READ 😄